packages/block-library/src/latest-posts/block.json

```diff
@@ -1,72 +1,75 @@
 {
-	"name": "core/latest-posts",
-	"category": "widgets",
-	"attributes": {
-		"align": {
-			"type": "string",
-			"enum":  ["left", "center", "right", "wide", "full"]
-		},
-		"className": {
-			"type": "string"
-		},
-		"categories": {
-			"type": "array"
-		},
-		"postsToShow": {
-			"type": "number",
-			"default": 5
-		},
-		"displayPostContent": {
-			"type": "boolean",
-			"default": false
-		},
-		"displayPostContentRadio": {
-			"type": "string",
-			"default": "excerpt"
-		},
-		"excerptLength": {
-			"type": "number",
-			"default": 55
-		},
-		"displayPostDate": {
-			"type": "boolean",
-			"default": false
-		},
-		"postLayout": {
-			"type": "string",
-			"default": "list"
-		},
-		"columns": {
-			"type": "number",
-			"default": 3
-		},
-		"order": {
-			"type": "string",
-			"default": "desc"
-		},
-		"orderBy": {
-			"type": "string",
-			"default": "date"
-		},
-		"displayFeaturedImage": {
-			"type": "boolean",
-			"default": false
-		},
-		"featuredImageAlign": {
-			"type": "string",
-			"enum": ["left", "center", "right"]
-		},
-		"featuredImageSizeSlug": {
-			"type": "string",
-			"default":"thumbnail"
-		},
-		"featuredImageSizeWidth": {
-			"type": "number",
-			"default":null
-		},
-		"featuredImageSizeHeight": {
-			"type": "number",
-			"default":null
-		}
-	}
+    "name": "core/latest-posts",
```

Comment:
It looks like this file has been reformatted and for no good reason. We should revert this change to only include 

```
 "postType": {
   "type": "string"
 },
```

It's probably an editor spaces setting issue :)